### PR TITLE
myfirstopensource

### PR DESCRIPTION
**Pull request by 1709abhishek**

### CODE REVIEW

File: javascript/answers/1709abhishek-q7.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <div>
+            <div>
+                <h4>
+                  What is the difference between == and ===?
+                </h4>
+            </div>
+            <div>
+                <br>
+                <strong>Answer:</strong>
+                <p> The == operator checks equality only whereas === checks equality, and data type, i.e., a value must be of the same type.</p>
```
Comment (dakshkhetan) [inline javascript/answers/1709abhishek-q7.html]: Please give an example to support your answer. :)

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Please run the command `find . -name '.DS_Store' -type f -delete` on terminal to delete those ".DS_Store" files which were created by your system.
And then commit your changes.

File: javascript/answers/1709abhishek-q7.html (hunk 1 above)
Comment (1709abhishek) [reply javascript/answers/1709abhishek-q7.html]: Screenshot 2020-02-17 at 1 14 44 PM
done.....please merge

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Please add an example to support your answer.

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**